<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tetris</class>
 <widget class="QMainWindow" name="Tetris">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>994</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tetris</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="TetrisScreen" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>871</width>
      <height>481</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>651</width>
       <height>331</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="frame">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QWidget" name="gridLayoutWidget_2">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>140</y>
           <width>61</width>
           <height>61</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="next_shape_board"/>
        </widget>
        <widget class="QLabel" name="next_shape_label">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>110</y>
           <width>121</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Segoe Script</family>
           <pointsize>10</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">text-align: center;</string>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Next Shape&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
        <widget class="QLabel" name="level_label">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>121</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Roboto</family>
           <pointsize>9</pointsize>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 1px solid white;
font: 9pt &quot;Roboto&quot;;
border-radius: 5px;</string>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; color:#ffffff;&quot;&gt;Level&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
        <widget class="QLabel" name="score_label">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>60</y>
           <width>121</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Roboto</family>
           <pointsize>9</pointsize>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 1px solid white;
font: 9pt &quot;Roboto&quot;;
border-radius: 5px;</string>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; color:#ffffff;&quot;&gt;Score&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
        <widget class="QLabel" name="level_output">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>121</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Cambria</family>
           <pointsize>10</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: white;</string>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#000000;&quot;&gt;0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
        <widget class="QLabel" name="score_output">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>90</y>
           <width>121</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Cambria</family>
           <pointsize>10</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: white;
</string>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#000000;&quot;&gt;0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
        <widget class="QLabel" name="command">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>190</y>
           <width>131</width>
           <height>131</height>
          </rect>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; text-decoration: underline; color:#ffffff;&quot;&gt;Command : &lt;/span&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;span style=&quot; font-weight:700; color:#ffffff;&quot;&gt;A&lt;/span&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt; : Rotate Left&lt;br/&gt;&lt;/span&gt;&lt;span style=&quot; font-weight:700; color:#ffffff;&quot;&gt;E&lt;/span&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt; : Rotate Right&lt;br/&gt;&lt;/span&gt;&lt;span style=&quot; font-weight:700; color:#ffffff;&quot;&gt;Q&lt;/span&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt; : Move Shape Left&lt;br/&gt;&lt;/span&gt;&lt;span style=&quot; font-weight:700; color:#ffffff;&quot;&gt;S&lt;/span&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt; : Move Shape Right&lt;br/&gt;&lt;/span&gt;&lt;span style=&quot; font-weight:700; color:#ffffff;&quot;&gt;D&lt;/span&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt; : Move Shape Down&lt;br/&gt;&lt;/span&gt;&lt;span style=&quot; font-weight:700; color:#ffffff;&quot;&gt;Space&lt;/span&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt; : Drop Shape&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="board">
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="TetrisGameOver" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>180</y>
      <width>301</width>
      <height>131</height>
     </rect>
    </property>
    <widget class="QLabel" name="test_over">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>0</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt; font-weight:700; color:#ffffff;&quot;&gt;You Lose&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="end_score">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>40</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;</string>
     </property>
     <property name="text">
      <string>Your score : </string>
     </property>
    </widget>
    <widget class="QLabel" name="end_score_output">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>40</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QPushButton" name="end_exit">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>90</y>
       <width>101</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Exit</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="backGround">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>991</width>
      <height>561</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>49</width>
      <height>16</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QFrame" name="MenuSCreen">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>40</y>
      <width>291</width>
      <height>391</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="Start">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>280</y>
       <width>101</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Exit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>360</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Exit</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Option">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>320</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Option</string>
     </property>
    </widget>
    <widget class="QLabel" name="tetris_title_label_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>211</width>
       <height>151</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe Print</family>
       <pointsize>20</pointsize>
       <bold>true</bold>
       <underline>true</underline>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/menumusic/Resources/images/Logo.ico);</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>331</width>
      <height>441</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <widget class="QWidget" name="formLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>311</width>
       <height>421</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QSpinBox" name="Box_Hauteur"/>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="Label_Hauteur">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Hauteur&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QSpinBox" name="Box_Largeur"/>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="Label_Largeur">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Largeur&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QSpinBox" name="Box_Niveau">
        <property name="wrapping">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="Label_Niveau">
        <property name="styleSheet">
         <string notr="true">color: white;</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Niveau&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="Label_Aleatoire">
        <property name="styleSheet">
         <string notr="true">border: 1px solid white;
color: white;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Generate randomly fulled table ?&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QRadioButton" name="random_board">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="BackMenu">
        <property name="text">
         <string>Back</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <zorder>backGround</zorder>
   <zorder>TetrisGameOver</zorder>
   <zorder>TetrisScreen</zorder>
   <zorder>MenuSCreen</zorder>
   <zorder>widget</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>994</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
